sdk/Javascript/src/Push/Push.FxOS.js

```diff
@@ -0,0 +1,413 @@
+// ----------------------------------------------------------------------------
```

Comment:
For this file, does FireFox OS need anything but mdn support? Alternatively, if native or Cordova based JS apps run on FireFox OS , it would be nicer to just add mdn into the Web.js version, and have both the FireFox and Web versions share this file.